amundsen_application/static/js/components/TableDetail/SourceLink/index.tsx

```diff
@@ -16,8 +16,22 @@ const SourceLink: React.FC<SourceLinkProps> = ({
 }: SourceLinkProps) => {
   if (tableSource === null || tableSource.source === null) return null;
 
-  const image =
-    tableSource.source_type === 'github' ? '/static/images/github.png' : '';
+  // const image =
+  //   tableSource.source_type === 'github' ? '/static/images/github.png' : '';
+
+  let image;
+  switch (tableSource.source_type) {
+    case 'github': {
+      image = '/static/images/github.png';
+      break;
+    }
+    case 'gql': {
+      image = '/static/images/gql.png';
+      break;
+    }
+    default:
+      image = '';
+  }
```

Comment:
Most of the times, switch statements are not recommended. Better off if we use a dictionary:
```
const sourceToImageURL = {
  'github': ...,
  'gql': ...,
};
const imageSRC = sourceToImageURL[tableSource.source_type] ? sourceToImageURL[tableSource.source_type] : '';
```

Reply:
Also, those source names should probably be part of an Enum, and do:
```
const sourceToImageURL = {
  [SOURCES.gitHub]: ...,
  [SOURCES.graphQL]: ...,
};